docs/pages/operator/init-manifests.mdx

```diff
@@ -1,8 +1,9 @@
 ---
 title: Applying manifests on initialization
-sidebar_label: Applying manifests on init
+sidebar_label: Applying manifests and charts on init
 ---
 
+## Applying manifests on initialization
 Starting in version 0.8.0, vcluster allows users to apply manifests as soon as the vcluster is started. This can be useful for users configuring internal vcluster resources.
```

Comment:
If this is accurate, I'd modify this from

> Starting in version 0.8.0, vcluster allows users to apply manifests as soon as the vcluster is started.

to

> Starting in version 0.8.0, vcluster allows users to apply manifests as soon as a virtual cluster is started.

It's a bit easier to distinguish the tool from the resource this way.